source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,181 @@
+
+#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+
+   For example:
+   https://download.clearlinux.org/releases/30010/clear/clear-30010-live-server.iso.xz
+
+#. Follow the instructions to :ref:bootable-usb based on your OS.
+
+#. Boot up the USB thumb drive.
+#. Select :guilabel:`Clear Linux OS` from the menu.
+#. In the console window, log in as root and set a password.
+#. Verify you have an IP address (network connection) and configure proxies (if needed).
+#. Download an :file:`live-server.yaml` template.
+
+   For example:
+
+   .. code-block:: bash
+
+      # curl -O https://download.clearlinux.org/releases/30010/clear/config/image/live-server.yaml
+
+#. Edit the template and change the settings as needed. Commonly changed settings include:
+
+   .. _install-configfile-yaml-begin:
+
+      * Under *block-devices*, set “file: "/dev/sda"” or enter your preferred device.
+      * Under *targetMedia*, set the third partition size to “0” to use the entire disk space.
+      * Under *bundles*, add additional bundles as needed.
+      * Delete the *post-install* section unless you have post-installation scripts.
+      * Under *Version*, set a version number. Use “0” for the latest version.
+
+      Commonly changed settings are shown in lines 15, 34, 37, and 51 below.
+      See `Installer YAML Syntax`_ for more details.
+
+   .. code-block:: bash
+      :linenos:
+      :emphasize-lines: 14,15,34,37,51
+
+   		#clear-linux-config
+
+   		# c-basic-offset: 2; tab-width: 2; indent-tabs-mode: nil
+   		# vi: set shiftwidth=2 tabstop=2 expandtab:
+   		# :indentSize=2:tabSize=2:noTabs=true:
+
+   		# File:         developer-live-server.yaml
+   		# Use Case:     Live Image which boots into login prompt
+   		#               Optionally allows for installing Clear Linux OS
+   		#               using the TUI clr-installer by running clr-installer
+
+   		# switch between aliases if you want to install to an actual block device
+   		# i.e /dev/sda
+   		block-devices: [
+   		   {name: "bdevice", file: "live-server.img"}
+   		]
+
+   		targetMedia:
+   		- name: ${bdevice}
+   		  type: disk
+   		  children:
+   		  - name: ${bdevice}1
+   		    fstype: vfat
+   		    mountpoint: /boot
+   		    size: "150M"
+   		    type: part
+   		  - name: ${bdevice}2
+   		    fstype: swap
+   		    size: "32M"
+   		    type: part
+   		  - name: ${bdevice}3
+   		    fstype: ext4
+   		    mountpoint: /
+   		    size: "3.0G"
+   		    type: part
+
+   		bundles: [os-core, os-core-update, NetworkManager, clr-installer, vim]
+
+   		autoUpdate: false
+   		postArchive: false
+   		postReboot: false
+   		telemetry: false
+   		iso: true
+   		keepImage: true
+   		autoUpdate: false
+
+   		keyboard: us
+   		language: en_US.UTF-8
+   		kernel: kernel-native
+
+   		version: 0
+
+   .. _install-configfile-yaml-end:
+
+#. Start the installation.
+
+   .. code-block:: bash
+
+   	  # clr-installer --config live-server.yaml

```

Comment:
Fix indentation that start with `clr-installer`--left justify to "c" in code block.  Please review and fix all throughout.